src/Compiler/Node.cpp

```diff
@@ -4,6 +4,20 @@
 
 namespace Ark::internal
 {
+
+    Node Node::TrueNode = Node("true");
+    Node Node::FalseNode = Node("false");
+    Node Node::NilNode = Node("nil");
+    Node Node::ListNode = Node("list");
+
+    void Node::init()
```

Comment:
Just a detail but you can mark it `noexcept` (in both the .cpp and .hpp) since it can not trigger exceptions